nemo/collections/tts/g2p/models/ctc.py

```diff
@@ -437,3 +439,58 @@ def per(self):
     @per.setter
     def per(self, per):
         self._per = per
+
+    # Methods for model exportability
+    def _prepare_for_export(self, **kwargs):
+        super()._prepare_for_export(**kwargs)
+
+        # Define input_types and output_types as required by export()
+        self._input_types = {
+            "input_ids": NeuralType(('B', 'T'), TokenIndex()),
+            "input_len": NeuralType(tuple('B'), LengthsType()),
+        }
+        self._output_types = {
+            # "preds_str": NeuralType(('B', 'T'), LabelsType()),
+            "log_probs": NeuralType(('B', 'T'), LossType()),
+            "encoded_len": NeuralType(('B', 'T'), LengthsType()),
+        }
+
+    def _export_teardown(self):
+        self._input_types = self._output_types = None
+
+    @property
+    def input_types(self):
+        return self._input_types
+
+    @property
+    def output_types(self):
+        return self._output_types
+
+    def input_example(self, max_batch=1, max_dim=44):
```

Comment:
It looks like `max_batch` and `max_dim` are not used. Are they required by the export code?